app/views/scheduled_pickups/_donation.html.erb

```diff
@@ -0,0 +1,4 @@
+<tr>
+  <td><%= donation.donor.name %></td>
+  <td><%= label_for_donation(donation) %></td>
```

Comment:
This is really in the weeds (as a recommendation), but could decorating the donation allow you to make these two attributes easier to populate?